changelog/28121.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+auth/okta: update to okta sdk v4
```

Comment:
It looks like this is okta sdk v5 now?

```suggestion
auth/okta: update to okta sdk v5
```

Reply:
good catch; v5 came out during development and the changes were cosmetic, so